portal/pom.xml

```diff
@@ -41,18 +41,48 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.tomcat</groupId>
+      <artifactId>tomcat-catalina</artifactId>
+      <version>7.0.68</version>
+      <scope>runtime</scope>
```

Comment:
Sounds good. It'll break Jetty, but I can handle that. As I hinted, I don't think we should use Tomcat for this anyway, as it locks the war to Tomcat. There are small and simple CORS filters we can use independently of the container.